### PR TITLE
feat: support go install

### PR DESCRIPTION
**Pull request by NeverRaR**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y
re #71  

#### 2. What is the scope of this PR (e.g. component or file name):

pkg/kclvm_runtime/kclvm.go
#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

support  install kclvm-go  through go install command

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by Peefy — COMMENTED** [review]
Ensure python3 and pip3 exist. Ref: https://github.com/KusionStack/KCLVM/blob/main/scripts/docker/kclvm/Dockerfile

File: pkg/tools/list/list_test.go [hunk 1]
```diff
@@ -79,6 +78,7 @@ func TestListDepFiles_failed(t *testing.T) {
 	}
 }
 
+/*
```
Comment (Peefy) [inline pkg/tools/list/list_test.go:83]: U can leave an issue number i.e. `https://github.com/KusionStack/kclvm-go/issues/74` directly.

**Review by Peefy — APPROVED** [review]
LGTM

**Review by zong-zhe — APPROVED** [review]
LGTM

**Pull request merged**